com.unity.render-pipelines.universal/Runtime/Decal/DBuffer/DBufferRenderPass.cs

```diff
@@ -135,7 +135,7 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
 
                 if (isDeferred)
                 {
-                    cmd.SetGlobalTexture("_CameraNormalsTexture", deferredLights.GbufferAttachmentIdentifiers[deferredLights.GBufferNormalSmoothnessIndex]);
+                    cmd.SetGlobalTexture("_CameraNormalsTexture", deferredLights.GbufferAttachments[deferredLights.GBufferNormalSmoothnessIndex].nameID);
```

Comment:
It's probably better to use the name directly
```
cmd.SetGlobalTexture(deferredLights.GbufferAttachments[deferredLights.GBufferNormalSmoothnessIndex].name, deferredLights.GbufferAttachments[deferredLights.GBufferNormalSmoothnessIndex].nameID);
```

Reply:
i think Decals rely on _CameraNormalsTexture being there, so we bind GBuffer2 as that. This would require changes for Decal shaders and probably additional checks whether it's forward or deferred